### DISCUSSION TITLE
How do I know I installed the identity pixel properly?

### DISCUSSION THREAD

**Discussion by jonathonbyrdziak** [merchantprotocol/theadslab#15] [User Guide Q&A] [answered]
So I've copied the installation snippet from my account and I've pasted it into the header of my website. Is this how I'm supposed to install this? Did I install the identity pixel properly? 

I just want to know for sure that it's tracking visitors and that the identity resolution is working.

**Comment by jonathonbyrdziak** [comment, marked as answer]
## Here's how you know for sure that you've installed it correctly.

This is a common question we get all the time, so we've built in some things to help you know for sure that the identity resolution is working on your new website.

If you've followed the process of copying the pixel from your account and pasting it into the header of your website, then yes, you've installed it correctly.

However, there are some situations where a minor detail is incorrect, causing the system to fail to fire and produce no analytics. After you've followed the copy/paste process here's the next steps to confirm a correct installation:

1. Open the website in your browser where the pixel is installed.

2. Make sure to clear all cache, even disabling cache temporarily can be helpful. You can do this by using incognito mode or through the Developer Tools Console if you're familiar with that. After clearing cache, refresh the page.
![Disable Cache](https://d1js66q00gh76r.cloudfront.net/b2v1%2Fpreview%2F45436547%2Fmain_full.png?response-content-disposition=inline%3Bfilename%3D%22main_full.png%22%3B&response-content-type=image%2Fpng&Expires=1648586459&Signature=eT2u~yx9wtHaqf0aas1wS75G65SrqcGGgy4GFg9TLLRagDmi75lrWE2pG~HVpgER3b~B6~2hAX~6ScugsD5WZL~-wFRmffG0lbOwiDNQC7eN1g0mRl~jg17g0RGAHE0L6QNANBgyVsf5QqeJ-os6rqbszYUZs1FhoK7t72rPEoxHEkYOpoeBowUHoXBpqKphMzoX9zYfKAfDNvkZE1gvJL9TKCaI6WCdVK5aCRvgbt6LSA-j9zqzh9R9r66I6CG3dCfm-w33TkVa2Ze2H4zK1m21YZ-eECTM-SaNQ1klSimAoMAsKiJUdMVeXXH3b~79ZuOm02wUaX0WHKHsn7AZvA__&Key-Pair-Id=APKAJT5WQLLEOADKLHBQ)

3. Open the Developer Tools Console (preferably in chrome).
![Right click to open developer tools](https://d1yffl6oyah1kr.cloudfront.net/34xi%2Ffile%2Fe138eb1e58996f8c8352bc1e2b075cb7_GIF+Recording+2022-03-29+at+1.33.25+PM.gif?response-content-disposition=inline%3Bfilename%3D%22GIF%20Recording%202022-03-29%20at%201.33.25%20PM.gif%22%3B&response-content-type=image%2Fgif&Expires=1648586622&Signature=ZK444ArW9gafffi~Ul9OjsymqkRtPh6tbCfcLD3Zz6548F5~iv0HoQpVw7i6VGHLEceCPQdpuWd8aFUH1hT3VzJc1F2EWU5su8~rz9lO4YCej9ywElw2s5WdPnhDAT8dQfA9KOFTxBZjQ6bNHnk9B~W7wJO5yrs9UL08LNqA0pEhyvA16A624RxRUC1~ibuFXoSM5Ls9h5MvQt0s~3uMFoIFBV8Sed6sdpRO2jESboBv9JdYQ3H9e6upggqbZfUe28u4NsNiyCjmpqVeoWnIZCp4ZRyrMwsQSBvFNykwcUubcrEYJdm2ePK9dJJjleuyLMhELWXfeJwOYAb19zKB6A__&Key-Pair-Id=APKAJT5WQLLEOADKLHBQ)

4. Navigate to the "Console" tab and you should see the following outputs of a successful pixel fire.
![Console output](https://d1l3wite3674hm.cloudfront.net/o9xi%2Ffile%2F60861247a364acc016f55d00bd5227b4_Screen+Shot+2022-03-29+at+1.36.35+PM.png?response-content-disposition=inline%3Bfilename%3D%22Screen%20Shot%202022-03-28%20at%208.15.30%20PM.png%22%3B&response-content-type=image%2Fpng&Expires=1648586820&Signature=A73OrM5gBO4L7OF4UPjvY3Ib46v1IqJeaQt06YKH5FO6Q3HNyDTQV36eArt2n8I~ReE9BkWd5nS3wFfRmka8I4H8OPn6tnaeggEasnKY5ctmwUBaTqReHtBTknj6lLOXHPOXKqj7gV0nHNJnPV8oaP9PiacuK3QHE4SNj9JIeXPeyJkvna2Hy670txyCyZC3wLvG4Ubupt7gA0B1FE1ciusROlGMJ9pb7403saX00nUr4Cvud0Zgul7BrwbC564TVvcoZdRNELCeYGiEIgPhOwV41YS86uKYWgOCWCmwLKxnz2ugaZYwsjmMesw6wr7fJpAJ~KsLaJ2ODtkg67mT5w__&Key-Pair-Id=APKAJT5WQLLEOADKLHBQ)

When you see any variation of console output that references The Ads Lab you will know for sure that the pixel is installed correctly on your website.

**Reply by Flywheeltargeting** [reply]
the images that you attached are not showing up
